operations/utility_storage/Add_Ceph_Node.md

```diff
@@ -30,10 +30,11 @@
     ```
 
     **IMPORTANT:** In the output from `watch ceph -s` the health should go to a `HEALTH_WARN` state. This is expected. Most commonly you will see an alert about `failed to probe daemons or devices`, but this should clear on its own.
+    In addition, it may take up to 5 minutes for the added OSDs to report as `up`.  This is dependent on the Ceph Orchestrator performing an inventory and completing batch processing to add the OSDs.
```

Comment:
```suggestion
    In addition, it may take up to 5 minutes for the added OSDs to report as `up`. This is dependent on the Ceph Orchestrator performing an inventory and completing batch processing to add the OSDs.
```